build/config/build.yaml

```diff
@@ -82,4 +87,61 @@ steps:
   dir: 'tests/'
   waitFor: ['DOCKER_STAGE', 'DOCKER_BUILD_MVN_GCLOUD']
 
+#
+# End of App Engine Integration tests
+
+#
+# GKE Integration tests
+#
+
+# Create a Docker image with the test application
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'DOCKER_BUILD_TEST_APPLICATION'
+  args: ['build', '--tag=${_STAGING_IMAGE}-test-app', '--no-cache', 'tests/runtimes-common-testing/target/deploy']
```

Comment:
Yes, it would definitely ease debugging to have a static workspace (only one build).

Is the first maven build enough (-P-local-docker-build and -P-test.local) to generate all of the requirement to run the smoke test  ?